### PR TITLE
feat: add SingleCollapse component

### PR DESCRIPTION
**Pull request by SimBig**

### CODE REVIEW

File: src/Collapse/index.tsx [hunk 1]
```diff
@@ -3,7 +3,28 @@ import {
...
 
 export const Collapse: typeof AntdCollapse = AntdCollapse;
 export type CollapseProps = AntdCollapseProps;
 export type CollapsePanelProps = AntdCollapsePanelProps;
+
+export function SingleCollapse(props: {
+  collapseProps?: Omit<CollapseProps, 'defaultActiveKey'> & {
+    defaultActive: boolean;
+  };
+  collapsePanelProps: Omit<CollapsePanelProps, 'key'>;
```
Comment (spawnia) [inline src/Collapse/index.tsx:16]:
```suggestion
  panelProps?: Omit<CollapsePanelProps, 'key'>;
```

File: src/Collapse/index.tsx [hunk 2]
```diff
@@ -3,7 +3,28 @@ import {
...
   CollapsePanelProps as AntdCollapsePanelProps,
 } from 'antd';
+import React, { ReactNode } from 'react';
 
 export const Collapse: typeof AntdCollapse = AntdCollapse;
 export type CollapseProps = AntdCollapseProps;
 export type CollapsePanelProps = AntdCollapsePanelProps;
+
+export function SingleCollapse(props: {
+  collapseProps?: Omit<CollapseProps, 'defaultActiveKey'> & {
```
Comment (spawnia) [inline src/Collapse/index.tsx:13]:
```suggestion
  collapseProps?: Omit<CollapseProps, 'activeKey' | 'defaultActiveKey'> & {
```

File: src/Collapse/index.tsx [hunk 3]
```diff
@@ -3,7 +3,28 @@ import {
...
+    defaultActive: boolean;
+  };
+  collapsePanelProps: Omit<CollapsePanelProps, 'key'>;
+  children: ReactNode;
+}) {
+  const key = 'default';
```
Comment (spawnia) [inline src/Collapse/index.tsx:19]: Has to be reinstantiated on every rerender, should rather be an `ALL_CAPS` constant defined outside of the component.

File: src/Collapse/index.tsx [hunk 4]
```diff
@@ -3,7 +3,28 @@ import {
...
 } from 'antd';
+import React, { ReactNode } from 'react';
 
 export const Collapse: typeof AntdCollapse = AntdCollapse;
 export type CollapseProps = AntdCollapseProps;
 export type CollapsePanelProps = AntdCollapsePanelProps;
+
+export function SingleCollapse(props: {
+  collapseProps?: Omit<CollapseProps, 'defaultActiveKey'> & {
+    defaultActive: boolean;
```
Comment (spawnia) [inline src/Collapse/index.tsx:14]:
```suggestion
    defaultActive?: boolean;
```

File: src/Timeline/index.tsx [hunk 5]
```diff
@@ -3,33 +3,7 @@ import {
...
 } from 'antd';
-import React, { ReactNode } from 'react';
-
-import { Collapse, CollapsePanelProps, CollapseProps } from '../Collapse';
 
 export const Timeline: typeof AntdTimeline = AntdTimeline;
 export type TimelineProps = AntdTimelineProps;
 export type TimelineItemProps = AntdTimelineItemProps;
-
-export function CollapsibleTimelineItem(props: {
```
Comment (spawnia) [inline src/Timeline/index.tsx:14]: Nice move towards composition.

File: src/Collapse/index.tsx [hunk 6]
```diff
@@ -3,7 +3,28 @@ import {
   CollapseProps as AntdCollapseProps,
   CollapsePanelProps as AntdCollapsePanelProps,
 } from 'antd';
+import React, { ReactNode } from 'react';
 
 export const Collapse: typeof AntdCollapse = AntdCollapse;
 export type CollapseProps = AntdCollapseProps;
 export type CollapsePanelProps = AntdCollapsePanelProps;
+
+export function SingleCollapse(props: {
```
Comment (spawnia) [inline src/Collapse/index.tsx:12]: `export type SingleCollapseProps = {...}`

**Review by spawnia — CHANGES_REQUESTED** [review]

**Comment by spawnia** [comment]
Commit message should contain a line with `BREAKING CHANGE`, since a component is removed. This will trigger the necessary major version bump.

**Comment by SimBig** [comment]
> Commit message should contain a line with `BREAKING CHANGE`, since a component is removed. This will trigger the necessary major version bump.


This should be solved handeld by the "!" after the "feat", shouldn't it?:
```
BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.
```
source: https://www.conventionalcommits.org/en/v1.0.0/

**Comment by spawnia** [comment]
`feat!: add SingleCollapse component` does not tell me what broke.

**Comment by SimBig** [comment]
> `feat!: add SingleCollapse component` does not tell me what broke.
That makes sense, of course. I have changed it accordingly

**Pull request merged**